src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,36 @@
+import styles from './Modal.module.scss';
+
+export default function Modal(props: any) {
+  return (
+    <>
+      <div className={styles.modalcontainer}>
+        <div className={styles.modal}>
+          <div className={styles.modalheader}>
+            <h3>Select your home location</h3>
```

Comment:
probably shouldn't be hardcoded (let's have it in props)

Reply:
good I agree

Reply:
fixed it